trading_ig/rest.py

```diff
@@ -512,17 +512,21 @@ def create_open_position(self, currency_code, direction, epic, expiry,
         else:
             raise IGException(response.text)
 
-    def update_open_position(self, limit_level, stop_level, trailing_stop,
-                             trailing_stop_distance, trailing_stop_increment,
-                             deal_id, session=None):
+    def update_open_position(self, limit_level, stop_level,
+                             deal_id, trailing_stop, trailing_stop_distance,
```

Comment:
`update_open_position` signature should be

    def update_open_position(...., trailing_stop=False, trailing_stop_distance=None, trailing_stop_increment=None):

it will avoid breaking existing code when no trailing stop is set.

Reply:
It's pull, copy and paste too fast :)

Reply:
Copy/paste and using Github website edit feature is a bad workflow (I'm pretty sure you also noticed that)...